siliconcompiler/tools/vivado/scripts/sc_bitstream.tcl

```diff
@@ -1,2 +1,17 @@
 open_checkpoint "inputs/${sc_topmodule}.dcp"
+
+if { [sc_cfg_tool_task_exists prescript] } {
```

Comment:
I think you can remove the `sc_cfg_tool_task_exists prescript` and `sc_cfg_tool_task_exists postscript` checks as they should not be needed